src/Fabulous.XamarinForms/Views/Layouts/FlexLayout.fs

```diff
@@ -0,0 +1,128 @@
+namespace Fabulous.XamarinForms
+
+open System.Runtime.CompilerServices
+open Fabulous
+open Xamarin.Forms
+
+type IFlexLayout =
+    inherit Fabulous.XamarinForms.ILayout
```

Comment:
FlexLayout (and other layouts I guess) actually inherits from ILayoutOfView (you need to create it though)